### PR TITLE
Publish distribution docs

### PR DESCRIPTION
**Pull request by moxiegirl**
- Fix for distribution headings and display of storage driver links
- Included some well regarded work people have been waiting for

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @docker/docs-owners Publish of distribution fixes and a few other things.

File: docs/sources/articles/configuring.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+page_title: Configuring Docker
+page_description: Configuring the Docker daemon on various distributions
+page_keywords: docker, daemon, configuration
+
+# Configuring Docker on various distributions
+
+After successfully installing the Docker daemon on a distribution, it runs with it's default
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: its

File: docs/sources/articles/configuring.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+config. Usually it is required to change the default config to meet one's personal requirements.
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Unhelpful passive voice. Docs use "you" (second person) for user.

File: docs/sources/articles/configuring.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ 
+Docker can be configured by passing the config flags to the daemon directly if the daemon
+is started directly. Usually that is not the case. A process manager (like SysVinit, Upstart,
+systemd, etc) is responsible for starting and running the daemon.
+
+Some common config options are
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: What commands are these flags associated with?

File: docs/sources/articles/configuring.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+page_description: Configuring the Docker daemon on various distributions
+page_keywords: docker, daemon, configuration
+
+# Configuring Docker on various distributions
+
+After successfully installing the Docker daemon on a distribution, it runs with it's default
+config. Usually it is required to change the default config to meet one's personal requirements.
+ 
+Docker can be configured by passing the config flags to the daemon directly if the daemon
+is started directly. Usually that is not the case. A process manager (like SysVinit, Upstart,
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Ambiguous pronoun antecedent ("that").

File: docs/sources/articles/configuring.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Some common config options are
+
+* `-D` : Enable debug mode
+
+* `-H` : Daemon socket(s) to connect to   
+
+* `--tls` : Enable or disable TLS authentication
+
+The complete list of flags can found at [Docker Command Line Reference](/reference/commandline/cli/)
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Missing period.

File: docs/sources/articles/configuring.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+    $ sudo stop docker
+
+    $ sudo restart docker
+
+
+### Configuring Docker
+
+Docker options can be configured by editing the file `/etc/default/docker`. If this file does not 
+exist, it needs to be createdThis file contains a variable named `DOCKER_OPTS`. All the
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Missing period, space.

File: docs/sources/articles/configuring.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+### Configuring Docker
+
+Docker options can be configured by editing the file `/etc/default/docker`. If this file does not 
+exist, it needs to be createdThis file contains a variable named `DOCKER_OPTS`. All the 
+config options need to be placed in this variable. For example
+
+    DOCKER_OPTS=" --dns 8.8.8.8 -D --tls=false -H tcp://0.0.0.0:2375 "
+
+The above daemon options :
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Extra space

File: docs/sources/articles/configuring.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    $ sudo stop docker
+
+    $ sudo restart docker
+
+
+### Configuring Docker
+
+Docker options can be configured by editing the file `/etc/default/docker`. If this file does not 
+exist, it needs to be createdThis file contains a variable named `DOCKER_OPTS`. All the 
+config options need to be placed in this variable. For example
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Missing comma or colon

File: docs/sources/articles/configuring.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Set dns server for all containers
+
+2. Enable Debug mode 
+
+3. Set tls to false
+
+4. Make the daemon listen for connections on `tcp://0.0.0.0:2375`
+
+After saving the file, restart docker using `sudo restart docker`. Verify that the daemon is
```
Comment (fredlf) [inline docs/sources/articles/configuring.md]: Docker. 

Many typos, not going to keep marking them. Please just give this a proof-read.

File: docs/sources/docker-hub/official_repos.md [hunk 10]
```diff
@@ -1,189 +1,106 @@
...
-
-Once you've checked off everything in these guidelines, and are confident your
-image is ready for primetime, please contact us at
-[partners@docker.com](mailto:partners@docker.com) to have your project
-considered for the Official Repos program.
+# Official Repositories on Docker Hub
+
+The Docker [Official Repositories](http://registry.hub.docker.com/official) are
+a curated set of Docker repositories that are promoted on Docker Hub and
+supported by Docker, Inc. They are designed to:
```
Comment (fredlf) [inline docs/sources/docker-hub/official_repos.md]: No, Official Repos are not supported by Docker, Inc. Check with Andy if you have questions.

**Comment by moxiegirl** [comment]
@fredlf  I'm sorry but this is not an appropriate venue for making edit to actual content. This material went through a review on the docker/distribution channel over a month ago. Please make a PR against that repository if you wish to make a change.

**Comment by jfrazelle** [comment]
LGTM

**Comment by moxiegirl** [comment]
Thanks Jess.

**Comment by fredlf** [comment]
I don't understand why my review is "inappropriate." This is an open PR with the status "needs docs review." I pointed out a factual error and a number of writing errors in the copy. I'm confused as to why you would dismiss that review feedback or why this venue is not the right one.

**Comment by moxiegirl** [comment]
@fredlf This is a cherry-pick into the **docs** branch. Everything in this PR has already been through a docs review in the **master* branch.

**Comment by fredlf** [comment]
Ah, now I understand. Did not divine that from the description, sorry. I would still hesitate before publishing something that claims we support Official Repos when in fact, we do not. This could have a negative impact on Andy's team, and mighta open us up to potential liability.

**Comment by SvenDowideit** [comment]
LGTM - @moxiegirl can you do me a favour next time, and list the PR's that are being merged into the docs branch?
when are you hitting publish?

**Comment by moxiegirl** [comment]
@fredlf The branch I am pulling is part of the PR information which GitHub creates automatically.

image

@SvenDowideit Next time. I'm publishing this tonight. The team has been waiting for it.

**Comment by thaJeztah** [comment]
I suggest to use a standard prefix in the title for this type of PRs, e.g., [Publish], [Merge] or [Docs Merge]. This will make it easier to see what kind of review is needed and hopefully prevents confusion (and frustration).

Good remarks by @fredlf btw, anyone has the opportunity to create a PR to fix that? (I'm currently unable to)

File: docs/sources/docker-hub/official_repos.md (hunk 10 above)
Comment (psftw) [inline docs/sources/docker-hub/official_repos.md]: Agree it was a mistake in language.  Docker, Inc supports the program, but does not actually provide support for each repository -- except in the cases where we don't have upstream maintaining them yet.
https://github.com/docker/docker/pull/12993

**Comment by moxiegirl** [comment]
@fredlf @SvenDowideit  Thanks for the catch Fred. I didn't publish last night. I sent a note to Peter to make the fix. Sorry for not hopping onto clarify until now but I had family in town last night (and this morning) so things were and are a bit hectic.

**Comment by fredlf** [comment]
@moxiegirl NP, glad it got fixed.

@thaJeztah, @moxiegirl  In the past, we have titled these kinds of PRs "Docs update #n". We number them so we know they're "in-between releases" updates and so that we have an idea of how many we've done. After a major release, the numbering restarts.

**Comment by moxiegirl** [comment]
@fredlf That sounds like a reasonable process. I was just going with what was written down in docs/README.  Why don't you update that file with this new process so we can review it in context?

**Pull request merged**